### PR TITLE
Daily task to remove old draft empty forms

### PR DESCRIPTION
**Pull request by erehulka**
Creates a task which runs daily and removed all forms (along with its files), which are:
1. Older than 1 week
2. Are in `DRAFT` state
3. Have `null` data

Tested locally and it works, however it should be watched on staging and the first run will contain many forms.

### CODE REVIEW

File: nest-forms-backend/src/forms/subservices/forms-task.subservice.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        formDataJson: {
+          equals: Prisma.DbNull,
+        },
+      },
+    })
+    const formIds = forms.map((form) => form.id)
+    const fileIds = forms.flatMap((form) => form.files.map((file) => file.id))
+
+    // Delete the forms along with its files
+    this.filesService.deleteFileMany(fileIds)
```
Comment (MarekBodingerBA) [inline nest-forms-backend/src/forms/subservices/forms-task.subservice.ts:43]: This probably needs to be awaited?

**Review by MarekBodingerBA — APPROVED** [review]
Perfect!

File: nest-forms-backend/src/forms/subservices/forms-task.subservice.ts (hunk 1 above)
Comment (MarekBodingerBA) [reply nest-forms-backend/src/forms/subservices/forms-task.subservice.ts:43]: There will still probably be empty folders left:
```
`/${formInfo.pospIdOrSlug}/${formInfo.formId}/`
```

I don't know if its something that we want to solve.

Comment (erehulka) [reply nest-forms-backend/src/forms/subservices/forms-task.subservice.ts:43]: Imo it is not necessary for now. I will create a followup issue.

Comment (erehulka) [reply nest-forms-backend/src/forms/subservices/forms-task.subservice.ts]: https://github.com/bratislava/konto.bratislava.sk/issues/1765

**Pull request merged**